src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speedmodes/other/MineBlazeHop.kt

```diff
@@ -17,6 +17,7 @@ object MineBlazeHop : SpeedMode("MineBlazeHop") {
                 mc.thePlayer.motionX *= 1.0015
                 mc.thePlayer.motionZ *= 1.0015
                 mc.timer.timerSpeed = 1.06f
+                mc.thePlayer.jump()
```

Comment:
ehm, if motion is above mc.thePlayer.motionY > 0.003 jump? I don't think this makes any sense and would cause the player to fly.